tests/test_unit/test_reports.py

```diff
@@ -4,28 +4,136 @@
 
 
 @pytest.mark.parametrize(
-    "data_selection",
+    "valid_dataset",
     [
-        lambda ds: ds.position,  # Entire dataset
-        lambda ds: ds.position.sel(
-            individuals="ind1"
-        ),  # Missing "individuals" dim
-        lambda ds: ds.position.sel(
-            keypoints="key1"
-        ),  # Missing "keypoints" dim
-        lambda ds: ds.position.sel(
-            individuals="ind1", keypoints="key1"
-        ),  # Missing "individuals" and "keypoints" dims
+        "valid_poses_dataset",
+        "valid_bboxes_dataset",
+        "valid_poses_dataset_with_nan",
+        "valid_bboxes_dataset_with_nan",
     ],
 )
-def test_report_nan_values(
-    capsys, valid_poses_dataset_with_nan, data_selection
+@pytest.mark.parametrize(
```

Comment:
I personally prefer to have the args in the functions signature in the same order as in the parametrisation, and things like capsys and request at the end.

Reply:
Same, I think `capsys` is actually a typo cause it's not used